fs/chunkedreader/chunkedreader.go

```diff
@@ -1,12 +1,18 @@
 package chunkedreader
```

Comment:
chunkedreader could do with a test!  Perphaps in a later commit?

Reply:
I added some tests in the latest version.
They are not testing all the features `ChunkedReader` has, but should ensure that reading and seeking yield correct results.